js/components/dataset/quality-section.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+<div>
+    <h3 class="pointer" :class="textClass" @click="toggle">
```

Comment:
Vue.js handle perfectly (now) these cases:
- take static classes from the component (if it's a component)
- merge static classes from outside the component (those declared on call)
- merge dynamic classes from the component
- merge dynamic classes from outside

So you can write something like:
```xml
<!-- my-component.vue -->
<template>
<div class="static-class-inside" :class="computedClasses"></div>
</template>
```
```xml
<!-- caller.vue -->
<template>
<my-component class="static-class-outside" :class="computedClassesOutside"></my-component>
</template>
```
And the 4 classes declaration will be merged !

Until now, I always split static and dynamic classes to enforce the difference, but if it's not that clear we can discuss and change that.